### DISCUSSION TITLE
Designing a ticket display section.

### DISCUSSION THREAD

**Discussion by Golden-Source** [filamentphp/filament#9543] [General]
We are facing difficulties in displaying tickets and their corresponding responses. We need assistance!

We have two tables named "tickets" and "ticket_messages."

In the "tickets" table, we store basic ticket information, such as ticket_id, user_id, title, department, status, created_at, and updated_at.

In the "ticket_messages" table, we save the responses that have been sent. For instance, a single ticket may have 15 messages sent by both admins and users. How can we display this information? The columns in the "ticket_messages" table include: id, ticket_id, user_id, message, is_read, is_admin, is_system, file_id, and ip_address.

We have been struggling with this for the past four days and have not been able to find a solution through demos and plugins. If anyone can assist us, we would greatly appreciate it.

**Comment by ArtMin96** [comment]
[Relation Managers](https://filamentphp.com/docs/3.x/panels/resources/relation-managers#creating-a-relation-manager)?

**Reply by Golden-Source** [reply]
![Untitled-3](https://github.com/filamentphp/filament/assets/50501111/a75ffbdc-e3f9-4082-88a7-c3895a5b6c65)

It seems like you want to achieve a specific style for the ticket section at the bottom of the image. However, without the image or a detailed description of the style you have in mind, it's challenging to provide specific guidance on how to implement it. If you could provide more details or describe the desired style in English, I'd be happy to help you with the code or steps needed to achieve that style.


![Untitled-4](https://github.com/filamentphp/filament/assets/50501111/dfe0a380-e022-4c07-8473-213cae39f0d8)
